autoreduce_qp/queue_processor/reduction/runner.py

```diff
@@ -114,6 +115,21 @@ def _get_mantid_version():
         return None
 
 
+def write_reduction_message(reduction, temp_output_file=None):
+    """
+    Write the reduction message to the reduction directory
+    """
+    if temp_output_file is None:
```

Comment:
I'm not sure there's a reason to try and maintain compatibility with `temp_output_file` since with this new structure that parameter will never be passed